md.obsidian.Obsidian.yml

```diff
@@ -0,0 +1,39 @@
+app-id: md.obsidian.Obsidian
+branch: master
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: obsidian.sh
+tags:
+  - proprietary
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+  - --share=network
+modules:
+  - name: obsidian
+    buildsystem: simple
+    build-commands:
+      - mkdir -p bin share/{appdata,applications,icons/hicolor/512x512/apps}
+      - mv md.obsidian.Obsidian.png share/icons/hicolor/512x512/apps/md.obsidian.Obsidian.png
+      - mv md.obsidian.Obsidian.desktop share/applications/md.obsidian.Obsidian.desktop
+      - mv obsidian.sh bin/obsidian.sh
+      - mv md.obsidian.Obsidian.appdata.xml share/appdata/md.obsidian.Obsidian.appdata.xml
+      - cp -r * ${FLATPAK_DEST}
+    sources:
+      - type: archive
+        url: https://github.com/obsidianmd/obsidian-releases/releases/download/v0.9.3/obsidian-0.9.3.tar.gz
```

Comment:
One thing I'd add here is `x-data-checker` as documented at https://github.com/flathub/flatpak-external-data-checker so Flathub version doesn't fall behind with releases, but that can be fixed after merge.